Automattic/wp-calypso
# Subscription Management: Add wpcom header for the external  path

Fixes #74071

## Proposed Changes

* Render the `UniversalNavbarHeader` for the logged-out layout of the `subscriptions` section
* Introduce an optional `variant` prop for the `UniversalNavbarHeader` component
  *  with the current allowed values of `default` and `minimal`
  * `default` - should display as it used to before the `variant` prop was introduced
  * `minimal` - should display the WordPress.com logo on the left hand side of the bar, and "Log In" as well as "Get Started" buttons on the right hand side of the bar

## Testing Instructions



* Open incgonito window (ensure you are logged out of wpcom)
* Go to http://calypso.localhost:3000/subscriptions

## Pre-merge Checklist




![Screen Shot 2023-03-08 at 12 15 38](https://user-images.githubusercontent.com/2019970/223701048-bc4feaf5-480b-4f39-984a-0b7f3302a835.png)
![Screen Shot 2023-03-08 at 12 15 50](https://user-images.githubusercontent.com/2019970/223701052-80e82502-d0d0-46d6-af15-86af965a1015.png)
